README.md

```diff
@@ -2230,26 +2232,69 @@ Please find below set of commands to create certificate, depending on OS.
-keytool -exportcert -alias datalab -storepass KEYSTORE_PASSWORD -file ~/keys/datalab.crt -keystore ~/keys/datalab.keystore.jks
-sudo keytool -importcert -trustcacerts -alias datalab -file ~/keys/datalab.crt -noprompt -storepass changeit -keystore ${JRE_HOME}/lib/security/cacerts
+keytool -genkeypair -alias ssn -keyalg RSA -storepass KEYSTORE_PASSWORD -keypass KEYSTORE_PASSWORD -keystore ~/keys/ssn.keystore.jks -keysize 2048 -dname "CN=localhost"
+keytool -exportcert -alias ssn -storepass KEYSTORE_PASSWORD -file ~/keys/ssn.crt -keystore ~/keys/ssn.keystore.jks
+sudo keytool -importcert -trustcacerts -alias ssn -file ~/keys/ssn.crt -noprompt -storepass changeit -keystore ${JRE_HOME}/lib/security/cacerts
 ```
 #### Create Windows server certificate
 
 Pay attention that the last command has to be executed with administrative permissions.
 To achieve this the command line (cmd) should be ran with administrative permissions.  
 ```
-"%JRE_HOME%\bin\keytool" -genkeypair -alias datalab -keyalg RSA -storepass KEYSTORE_PASSWORD -keypass KEYSTORE_PASSWORD -keystore <DRIVE_LETTER>:\home\%USERNAME%\keys\datalab.keystore.jks -keysize 2048 -dname "CN=localhost"
-"%JRE_HOME%\bin\keytool" -exportcert -alias datalab -storepass KEYSTORE_PASSWORD -file <DRIVE_LETTER>:\home\%USERNAME%\keys\datalab.crt -keystore <DRIVE_LETTER>:\home\%USERNAME%\keys\datalab.keystore.jks
-"%JRE_HOME%\bin\keytool" -importcert -trustcacerts -alias datalab -file <DRIVE_LETTER>:\home\%USERNAME%\keys\datalab.crt -noprompt -storepass changeit -keystore "%JRE_HOME%\lib\security\cacerts"
+"%JRE_HOME%\bin\keytool" -genkeypair -alias ssn -keyalg RSA -storepass KEYSTORE_PASSWORD -keypass KEYSTORE_PASSWORD -keystore <DRIVE_LETTER>:\home\%USERNAME%\keys\ssn.keystore.jks -keysize 2048 -dname "CN=localhost"
+"%JRE_HOME%\bin\keytool" -exportcert -alias ssn -storepass KEYSTORE_PASSWORD -file <DRIVE_LETTER>:\home\%USERNAME%\keys\ssn.crt -keystore <DRIVE_LETTER>:\home\%USERNAME%\keys\ssn.keystore.jks
+"%JRE_HOME%\bin\keytool" -importcert -trustcacerts -alias ssn -file <DRIVE_LETTER>:\home\%USERNAME%\keys\ssn.crt -noprompt -storepass changeit -keystore "%JRE_HOME%\lib\security\cacerts"
 
 Useful command
-"%JRE_HOME%\bin\keytool" -list -alias datalab -storepass changeit -keystore "%JRE_HOME%\lib\security\cacerts"
-"%JRE_HOME%\bin\keytool" -delete -alias datalab -storepass changeit -keystore "%JRE_HOME%\lib\security\cacerts"
+"%JRE_HOME%\bin\keytool" -list -alias ssn -storepass changeit -keystore "%JRE_HOME%\lib\security\cacerts"
+"%JRE_HOME%\bin\keytool" -delete -alias ssn -storepass changeit -keystore "%JRE_HOME%\lib\security\cacerts"
 ```
 Where the ```<DRIVE_LETTER>``` must be the drive letter where you run the DataLab.
 
+### Set up config files
 
+Add those  file to .git/info/exclude:
+* infrastructure-provisioning/src/ssn/templates/ssn.yml
+
+* services/self-service/self-service.yml
+
+* services/provisioning-service/provisioning.yml
```

Comment:
Sometimes we do need to change these files and push it to the remote. I would not recommend describing this section.

Reply:
Ok, removed.